assets/scss/modules/_datatables.scss

```diff
@@ -2,6 +2,7 @@
   position: relative;
   display: inline-block;
   width: 100%;
+  margin-top: em(50);
```

Comment:
dataTables_* are class names added by the plugin, including this one for dataTables_wrapper.